### ISSUE TITLE
Create Landing Page and Data Portal Coming Soon Page for LandPKS on staging

### ISSUE THREAD

**Issue by DerekCaelin** [techmatters/terraso-product#814]
Use [Draft Language MVP Landing Page](https://docs.google.com/document/d/1J0IHP4gHxakHcvHKDD6qNMOyrk7HRasjpvUdrrU5cDg/edit#heading=h.q0dns7qjtkun)

- [x] find theme
- [x] create styling and theme for MVP page
- [x] build content for MVP page
- [x] Create data portal "coming soon" page at address `https://landpks.terraso.org/portal`

this will be the basis for: https://github.com/techmatters/terraso-product/issues/829

[Logo files](https://drive.google.com/drive/folders/11oSPm0bzJxbmqgQ0fmLTOmEUYWPvW449?usp=drive_link) 

[Homepage]( https://landpks-terraso-staging.mystagingwebsite.com/)
[Data Portal page](https://landpks-terraso-staging.mystagingwebsite.com/landpks-data-portal/)

**Comment by CourtneyLee333** [comment]
@paulschreiber Can you help me with adding a simple form to the "Get News and Updates" section? Ideally it would be a single field for email address and the submit button.

**Comment by CourtneyLee333** [comment]
@DerekCaelin Will you please create an issue to improve the Contact Us forms experience?

**Comment by DerekCaelin** [comment]
@CourtneyLee333 any thoughts on what requirements you'd like reflected in the issue?

**Comment by CourtneyLee333** [comment]
@DerekCaelin Not much yet, need to think it through. But currently there are simply buttons that lead to generic unbranded standalone forms, which is a jarring. Part of the issue should be to research and plan the solution.

**Comment by DerekCaelin** [comment]
Here is an issue - since both the "contact" and "support" forms are without branding, i listed them both for a revisit/review: https://github.com/techmatters/terraso-product/issues/866

**Comment by CourtneyLee333** [comment]
@DerekCaelin I'm assigning this to you because the landing page is done except for all of the logos that need to go under the Partners and Collaborators section. I've got the grid set up and styled, so it's just a matter of someone putting the logos in. We prefer svg whenever possible. I added a link to the folder in Drive where the logos live, but I've had to go searching online for svgs sometimes.
